### DISCUSSION TITLE
How to clear tox cache?

### DISCUSSION THREAD

**Discussion by SCSR-is-too-short-username** [tox-dev/tox#3213] [Q&A] [answered]
I have an issue where tox caches on older version of a package and the tests fail because of that. The newer version has changed a function name. How do I clear the cache so it downloads the updated version?

**Comment by jugmac00** [comment, marked as answer]
`tox -r <envname>` should help in that case. If not, you could simply delete the `.tox` folder - there are only transient files in there, so this just rebuilds everything.

If `tox -r` does not help, please report back, then we probably need to add some documentation.

**Comment by SCSR-is-too-short-username** [comment]
Using `tox --recreate` seemed to work for me
